tex/latex/pgf/doc/pgfmanual-en-macros.tex

```diff
@@ -1101,6 +1101,21 @@
   \end{pgfmanualentry}
 }
 
+\newenvironment{pgfkeyslibrary}[1]{%
+  \begin{pgfmanualentry}%
+    \pgfmanualentryheadline{%
+      \pgfmanualpdflabel{#1}{}%
+      \textbf{\texttt{pgfkeys} Library} \texttt{\declare{#1}}}%
+    \index{#1@\protect\texttt{#1} library}%
+    \index{Libraries!#1@\protect\texttt{#1}}%
```

Comment:
Does it need to be distinguished with tikz libraries? If so
```suggestion
    \index{#1@\protect\texttt{#1} \protect\texttt{pgfkeys} library}%
    \index{pgfkeys Libraries@\protect\texttt{pgfkeys} Libraries!#1@\protect\texttt{#1}}%
```

Reply:
There is no distinction between PGF and TikZ libraries already. What do you think?

Reply:
But `\pgfuselibrary` is never documented nor mentioned in `pgfmanual`. That means currently all the "xxx library" index entry in `pgfmanual` is actually a tikz library.